ebuild-maintenance/git/text.xml

```diff
@@ -286,16 +283,6 @@ are optionally used in Gentoo:
     referenced, it will be closed (as fixed) automatically with reference
     to the commit.
   </li>
-  <li>
-    <c>Package-Manager:</c> This is automatically inserted by
-    <c>repoman commit</c> and it specifies the version of
-    <c>sys-apps/portage</c> on the system.
-  </li>
-  <li>
-    <c>RepoMan-Options:</c> This is automatically inserted by
-    <c>repoman commit</c> and records the options passed to repoman (such as
-    --force) for the commit.
-  </li>
```

Comment:
This describes where these tags originate from, as they are present in existing commits. I suggest to keep both items.

Reply:
I'm ambivalent, slightly leaning towards removing them.

If no one is using `repoman`, this only serves to document what the tags in git history mean -- which seem entirely self-explanatory. I don't see any value in keeping them in the devmanual.

If you feel that strongly about keeping them, I guess I could change their wording to use the past tense, e.g. instead of "This *is* automatically inserted by repoman commit" → "This *was* automatically inserted by repoman commit" to make it clear that repoman commit is the old, deprecated tool.

Reply:
Changed to use the past tense.